hermes/probe/delete/delete.go

```diff
@@ -0,0 +1,133 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Delete_file implements the probe operation for deleting a file in a
+// storage system.
+
+// Package delete implements the file deletion operation with a storage system.
+package delete
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"time"
+
+	"cloud.google.com/go/storage"
+	"github.com/google/cloudprober/logger"
+	"github.com/googleapis/google-cloud-go-testing/storage/stiface"
+	"github.com/googleinterns/step224-2020/hermes/probe"
+	"github.com/googleinterns/step224-2020/hermes/probe/metrics"
+	"google.golang.org/api/iterator"
+
+	pb "github.com/googleinterns/step224-2020/hermes/proto"
+)
+
+const (
+	apiLatency             = "hermes_api_latency_seconds" // TODO(evanSpendlove) add this constant to metrics.go
+	minFileIDToDelete      = 11                           // we can delete files starting from the file Hermes_11
+	numberOfDeletableFiles = 40                           // there are 40 files to delete from [Hermes_11,Hermes_50]
```

Comment:
Just in case you missed leczb's comment. ;)
```suggestion
	maxFileIDToDelete = 40                           // there are 40 files to delete from [Hermes_11,Hermes_50]
```

Reply:
Thanks :) I've made this change in a commit.